### PR TITLE
Creating PR to start working on UI of login page

### LINKED ISSUE #32: Create login page UI

### PR DESCRIPTION
**Pull request by Maher9596**
# start working on login page

### CODE REVIEW

**Comment by Maher9596** [comment]
# Login page UI
- I styled the UI for the login page using my own styling + Font awesome library
- I added a **Forgot password** link (We can discuss and decide whether we want to keep it there or not)
# Issues
- I faced this issue while trying to connect my CSS file to my ejs file I got the following error

image

It seemed to me that **Express** did not know where to find my CSS file, So I did my research and found out that we need to specify where can express find all of our static files, And from **Express documentation** I found that we can do that by adding this line of code.
image

File: public/login.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: radial-gradient(#9b30ff, #9225a7, #7f2092);
```
Comment (Ma7eer) [inline public/login.css:4]: All our colors should be in variables. read about [css variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).

File: public/login.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  height: 100vh;
+  overflow: hidden;
+  font-family: "Noto Sans TC", sans-serif;
+}
+.center {
+  width: 430px;
```
Comment (Ma7eer) [inline public/login.css:10]: No [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)).

File: views/pages/login.ejs [hunk 3]
```diff
@@ -1,7 +1,40 @@
...
+                <input type="text" placeholder="Email">
+                <i class="far fa-envelope"></i> 
+                <input id="pswrd" type="password" pattern=".{6}" required title="6 characters minimum" placeholder="Password">
+                <i class="fas fa-lock" onclick="show()"></i>
+                <input id="btn" type="submit" value="Sign in">
+                <a href="#">Forgot Password</a>
+            </form>
+        </div>
+        <script>
+            const pswrd = document.getElementById('pswrd')
```
Comment (Ma7eer) [inline views/pages/login.ejs]:
```suggestion
            const password = document.getElementById('pswrd')
```

File: views/pages/login.ejs [hunk 4]
```diff
@@ -1,7 +1,40 @@
...
+                pswrd.style.marginTop = "20px"
+                icon.style.color - "#7f2092"
+            }else{
+                pswrd.type = "password"
+                icon.style.color="grey"
+            }
+        </script>
+    </body>
+</html>
+
```
Comment (Ma7eer) [inline views/pages/login.ejs]:
```suggestion

```

**Review by Ma7eer — CHANGES_REQUESTED** [review]
I added some comments to help make the code more readable. 

In addition, see below for some comments on the UI:

- I don't like the color of the font. Lets stick to our main colors: purple,grey, white
- The horizontal line below the title is unnecessary.
- The gradient is currently going to the center lets add some [angles](https://www.quirksmode.org/css/images/angles.html) to it.  
- The form is too low. it should be on the center vertically and horizontally.
- Remove the forget password link as we won't be adding that functionality.

**Pull request closed without merging**